distribution/lib/Standard/Database/0.0.0-dev/src/DB_Table.enso

```diff
@@ -206,8 +207,12 @@ type DB_Table
        dropped from the output.
 
        Arguments:
-       - columns: Specifies columns by a single instance or Vector of names,
-         indexes or regular expressions to match names.
+       - columns: Specifies columns by a single instance or Vector of names;
```

Comment:
I wonder if this might read better if we listed the different kinds of valid arguments as a nested list, and (in the case of `By_Type`, arguments to an argument as a futher nested list.

Reply:
I can't get a nested list to render correctly...